### PR TITLE
FHAC-324:Fix CONNECT to Support PKCS KeyStoreType

### PR DESCRIPTION
**Pull request by alameluchidambaram**
1. Add KeyStore check for "JKS" and "PKCS11"

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/CertificateManagerImpl.java [hunk 1]
```diff
@@ -146,8 +150,15 @@ private void initKeyStore() throws Exception {
         } else {
             try {
                 keyStore = KeyStore.getInstance(storeType);
-                is = new FileInputStream(storeLoc);
-                keyStore.load(is, password.toCharArray());
+                if (JKS_TYPE.equalsIgnoreCase(storeType)) {
+                    is = new FileInputStream(storeLoc);
+                    keyStore.load(is, password.toCharArray());
+                } else if (PKCS_TYPE.equalsIgnoreCase(storeType)) {
+                    keyStore.load(is, password.toCharArray());
```
Comment (jsmith2) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/CertificateManagerImpl.java]: It's ok to pass a null input stream here?

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/CertificateManagerImpl.java [hunk 2]
```diff
@@ -146,8 +150,15 @@ private void initKeyStore() throws Exception {
         } else {
             try {
                 keyStore = KeyStore.getInstance(storeType);
-                is = new FileInputStream(storeLoc);
-                keyStore.load(is, password.toCharArray());
+                if (JKS_TYPE.equalsIgnoreCase(storeType)) {
```
Comment (nareshcgi) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/CertificateManagerImpl.java]: We need the same logic for trust store also.

**Comment by nareshcgi** [comment]
Reviewed and good with the PR.

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without comment.

**Pull request merged**